be/src/docker-compose.override.yml

```diff
@@ -160,7 +160,7 @@ services:
   basket-api:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
-      - ConnectionString=basketdata
+      - RedisConnectionString=basketdata
```

Comment:
**suggestion:** name it back to ConnectionString please. Otherwise deployment will stop working.